Document/0x06a-Platform-Overview.md

```diff
@@ -48,35 +48,33 @@ Signing application code in iOS is different than in Android. In the latter you
+The [app sandbox](https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html "File System Basics") is an access control technology provided for iOS and enforced at kernel level. Its purpose is to limit the impact and damage to the system and user data that may occur when an app is compromised.
 
-In line with the "crystal prison" theme, sandboxing has been a core security feature since the first releases of iOS. As a principle, all user applications run under the same user `mobile`, with only a few system applications and services running as `root`. Regular apps on iOS are confined to a "container" that restrict access to the app's own files and a very limited amount of system APIs. Access to all resources, like files, network sockets, IPCs, shared memory, etc. will be then controlled by the sandbox. Restrictions include [#levin]:
+Along with the "crystal prison”, sandboxing has been a core security feature since the first releases of iOS. As a principle, all user applications run under the same user `mobile`, with only a few system applications and services running as `root`. Regular apps on iOS are confined to a "container" that restricts access to the app's own files and a very limited amount of system APIs. Access to all resources, like files, network sockets, IPCs, shared memory, etc. will be then controlled by the sandbox. These restrictions work the following ways.
 
-- The app process is restricted to it's own directory (under /var/mobile/Containers/Bundle/Application/) using a chroot-like mechanism.
-- The mmap and mmprotect() system calls are modified to prevent apps from make writeable memory pages executable and preventing processes from executing dynamically generated code. In combination with code signing and FairPlay, this enforces strict limitations on what code can be run under specific circumstances (e.g., all code in apps distributed via the app store is approved by Apple).
-- Isolation from other running processes, even if they are owned by the same UID.
-- Hardware drivers cannot be accessed directly. Instead, any access goes through Apple's frameworks.
+- App process is restricted to its own directory (under /var/mobile/Containers/Bundle/Application/) using a chroot-like mechanism.
+- Mmap and mmprotect() system calls are modified to prevent apps from making writeable memory pages executable and preventing processes from executing dynamically generated code. In combination with the code signing and FairPlay, this enforces strict limitations on what code can be run under specific circumstances (e.g., all code in apps distributed via the app store is approved by Apple).

```

Comment:
> Mmap and mmprotect() system calls

I think this should still be "The ... system calls"

> In combination with the code signing

Code signing is a proces, so it isn't "the code signing".